functionary/core/models/workflow_parameter.py

```diff
@@ -0,0 +1,49 @@
+from django.core.validators import RegexValidator
+from django.db import models
+
+PARAMETER_TYPES = [
```

Comment:
Yeah, I originally had a TODO here to find a better home for this where it would make sense to reuse it between the two.  I briefly had it in the Function model, which feels like a better home, except that it isn't actually used by Function, so that felt weird.  If a `FunctionParameter` model were a thing, that would definitely be the right spot.  Maybe that should be a thing...That's a whole 'nother conversation though.